### ISSUE TITLE
m43aa.git not found

### ISSUE THREAD

**Issue by VlntKnight** [KOTORCommunityPatches/K1_Community_Patch#386]
Fairly new to modding KOTOR, but have modded other games in the past, so sorry in advance for any mistakes or errors I make.

I did a clean install of KOTOR, and started modding, following the instructions detailed in [this](https://www.reddit.com/r/kotor/wiki/kotormodbuildfull#wiki_kotor_1_full_build) Reddit post on r/kotor. I went down the list installing the mods in the order listed, but when it comes to installing the community patch with the installer, it always hangs at file m43aa.git, saying the system could not find it.

I went and checked the .rar archive the installer comes from with WinRAR, and discovered that m43aa.git was not within it, nor any of the folders within. I've tried re-downloading the file from Deadly Stream, but haven't been able to get ahold of this missing file.

**Comment by DarthParametric** [comment]
It's not missing. A physical file is not required. It edits the GIT inside unk_m43aa.mod. The installation works for everyone else, so there's some particular issue on your end causing the problem. 

You need to provide the following. Your OS, the full path of the game, the full path of where you extracted the K1CP archive, and a copy of the installation log the K1CP installer produces.

**Comment by VlntKnight** [comment]
Ahh, apologies. I didn't realise that was how it worked. It has also occured to me that perhaps Github was not the place for this kind of issue, and I should have tried asking for help in r/KOTOR first, so sorry about that.


I am on Windows 10 Pro, version 1903. My OS build is 18362.778.

KOTOR is installed in the path X:\Steam\steamapps\common\swkotor where drive X is a Samsung SSD being used as my second drive (I only have 3 don't let the strange drive letter fool you). I pointed the archive towards my swkotor folder, so I hope that was the right place.

I attempted the installation once more this morning, and ran into the same issue on the same line, so I provided the log in what I hope is a working link to a .txt file.

[K1CP Log.txt](https://github.com/KOTORCommunityPatches/K1_Community_Patch/files/4575625/K1CP.Log.txt)


Again, apologies for the inconvenience, and thank you for your assistance thus far.
